scipy/_lib/_uarray/_backend.py

```diff
@@ -186,7 +191,7 @@ def generate_multimethod(
         return an (args, kwargs) pair with the dispatchables replaced inside the args/kwargs.
     domain : str
         A string value indicating the domain of this multimethod.
-    default : Optional[Callable], optional
+    default: Optional[Callable], optional
```

Comment:
IIRC we recently got some PR to add such spaces. Looking at numpydoc spaces are present.

Reply:
That's not really interesting here, since this is not part of our docs. It could be changed upstream in Uarray, but it doesn't really matter much either way - we probably have better things to do.